providers/azureblob/src/main/java/org/jclouds/azureblob/blobstore/AzureBlobStore.java

```diff
@@ -368,7 +368,7 @@ public void setContainerAccess(String container, ContainerAccess access) {
 
    @Override
    public BlobAccess getBlobAccess(String container, String key) {
-      throw new UnsupportedOperationException("unsupported in Azure");
+      return BlobAccess.PRIVATE;
```

Comment:
Done for `AzureBlobStore.getBlobAccess` and `RegionScopedSwiftBlobStore.getBlobAccess`. Will fix `GlacierBlobStore.getBlobAccess` [here](https://github.com/jclouds/jclouds-labs-aws/blob/master/glacier/src/main/java/org/jclouds/glacier/blobstore/GlacierBlobStore.java)